### DISCUSSION TITLE
Argo Rollouts for GKE Multi-Cluster Setup

### DISCUSSION THREAD

**Discussion by n0rm4l-me** [argoproj/argo-rollouts#3528] [Q&A]
Currently I have a Multi-Cluster setup as following: 

1. Config Cluster.
3. East Cluster.
4. West Cluster.

MC Gateway and HTTPRoutes are stored in Config Cluster, while applications are running in East and West clusters. Is it possible to use Canary / Blue-Green and modify HTTPRoutes in different cluster with Argo Rollouts?
I tried the Gateway API plugin, it is throwing error saying HTTPRoute not found (obviously it is in different cluster).
Any solutions for GKE Multi-cluster setup?

**Comment by kostis-codefresh** [comment]
> Is it possible to use Canary / Blue-Green and modify HTTPRoutes in different cluster with Argo Rollouts?

From the newly added best practices  https://argo-rollouts.readthedocs.io/en/latest/best-practices/#understand-the-scope-of-argo-rollouts

"Currently Argo Rollouts works with a single Kubernetes deployment/application and within a single cluster only."

There are several open Github issues for discussion. Please upvote/comment on the one that matches closer your use case.
